packages/conventional-recommended-bump/src/bumper.spec.ts

```diff
@@ -56,57 +56,73 @@ describe('conventional-recommended-bump', () => {
     it('recommends major if ! is used in isolation', async () => {
       preparing(2)
 
-      const bumper = new Bumper(testTools.cwd).loadPreset('conventionalcommits')
+      const bumper = new Bumper(testTools.cwd).loadPreset(
+        'conventionalcommits'
+      )
```

Comment:
Please reduce unneeded diff  🙏

Reply:
Yes! Sorry about that - auto formatting from editor...  I will clean that up!

Reply:
@dangreen Should be good now